client/js/src/client.ts

```diff
@@ -259,7 +259,7 @@ export function api_factory(
 				const ws = await import("ws");
 				NodeBlob = (await import("node:buffer")).Blob;
 				//@ts-ignore
-				global.WebSocket = ws.WebSocket;
+				if (!global.WebSocket) global.WebSocket = ws.WebSocket;
```

Comment:
Could we live this check to line 258 instead? If we don't need the poly fill then we shouldn't bother loading it in the first place.

Reply:
Good call. Changed 👍